packages/firestore/src/api/geo_point.ts

```diff
@@ -23,15 +23,24 @@ import {
 import { primitiveComparator } from '../util/misc';
 
 /**
- * Immutable class representing a geo point as latitude-longitude pair.
- * This class is directly exposed in the public API, including its constructor.
+ * An immutable object representing a geo point in Firestore. The geo point
+ * is represented as latitude/longitude pair.
+ *
+ * Latitude values are in the range of [-90, 90].
+ * Longitude values are in the range of [-180, 180].
  */
 export class GeoPoint {
   // Prefix with underscore to signal this is a private variable in JS and
   // prevent it showing up for autocompletion when typing latitude or longitude.
   private _lat: number;
   private _long: number;
 
+  /**
+   * Creates a new immutable GeoPoint object with the provided latitude and
```

Comment:
GeoPoint in backticks?

Also, should these literal numeric values be in backticks?